server/controller/FlightController.ts

```diff
@@ -542,15 +554,14 @@ function paramIdIsLeonardo(req, res) {
 }
 
 function createMulterIgcUploadHandler({ parts = 1 } = {}) {
+  const dataPath = process.env.SERVER_DATA_PATH;
+  if (!dataPath) throw new Error("SERVER_DATA_PATH not set");
```

Comment:
All env's and there defaults are defined in env-config.ts.
This here was missed to replace.

Remove the check and use env-config